internal/proto/milvus.proto

```diff
@@ -141,26 +141,49 @@ message DescribeCollectionResponse {
   int32 shards_num = 8; 
 }
 
+/**
+* Load collection data into cache, then you can vector search by this collection.
```

Comment:
"Load collection data into query nodes" may be better.

Reply:
And we can mention that a collection cannot be searched if this collection was not loaded.

Reply:
> And we can mention that a collection cannot be searched if this collection was not loaded.

I think user can realize that.